web/assets/javascripts/dashboard-charts.js

```diff
@@ -86,6 +86,7 @@ class RealtimeChart extends DashboardChart {
     updateStatsSummary(this.stats.sidekiq);
     updateRedisStats(this.stats.redis);
     updateFooterUTCTime(this.stats.server_utc_time);
+    updateNumbers();
```

Comment:
This is in application.js. Is it needed here?

Reply:
Yeah, it has to run again after `updateStatsSummary` because the numbers get overwritten by polling mechanism.